lib/variable.js

```diff
@@ -8,7 +8,7 @@ const base64Regex = /^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-
  * the variable you wish to have returned
  * @return {Object}
  */
-module.exports = function getVariableAccessors (container, varName, defValue) {
+module.exports = function getVariableAccessors (container, varName, defValue, extraAccessors = {}) {
```

Comment:
@todofixthis I think removing the default param here will fix the other build issue with code coverage.